webvtt.go

```diff
@@ -169,6 +170,9 @@ func ReadFromWebVTT(i io.Reader) (o *Subtitles, err error) {
 				strings.HasSuffix(webVTTStyles.WebVTTStyles[len(webVTTStyles.WebVTTStyles)-1], "}") {
 				blockName = ""
 			}
+			// Reset tag stack
```

Comment:
Can you add a new line before the comment and change the comment to `// Reset tags`?